### PR TITLE
CNS-310: organize specs APIs

### PR DESCRIPTION
**Pull request by oren-lava**
* **Please check if the PR fulfills these requirements**
- [ ] Commit message follows the Contribution Guidelines
- [ ] Tests ran locally and added/modified if needed
- [ ] Docs have been added/updated, if applicable
- [ ] If applicable - JIRA ticket ID was added

* **What kind of change does this PR introduce?** (Bug fix, feature, unit tests, docs update, ...)
specs update

* **What is the current behavior?** (You can also link to an open issue here)
specs were not sorted and had missing APIs

* **What is the new behavior (if this is a feature change)?**
fix the issue

* **Please describe what manual tests you ran, if applicable**

* **Other information**:

### CODE REVIEW

File: cookbook/spec_add_cosmossdk_full.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "title": "Add Specs: Full Cosmos SDK",
+        "description": "Adding full Cosmos SDK specification",
+        "specs": [
+            {
+                "index": "COSMOSSDKFULL",
+                "name": "Full Cosmos SDK",
+                "enabled": false,
+                "imports": [
+                    "COSMOSSDK",
+                    "COSMOSWASM"
```
Comment (SeanZoR) [inline cookbook/spec_add_cosmossdk_full.json:12]: 👀 great catch

File: cookbook/spec_add_ibc.json [hunk 2]
```diff
@@ -0,0 +1,1734 @@
+{
+    "proposal": {
+        "title": "Add Specs: IBC",
```
[the remark this answers is not in the document]
Comment (oren-lava) [reply cookbook/spec_add_ibc.json:3]: thanks

File: cookbook/spec_add_cosmossdk_full.json (hunk 1 above)
Comment (oren-lava) [reply cookbook/spec_add_cosmossdk_full.json:12]: Thanks :)

File: cookbook/spec_add_lava.json [hunk 3]
```diff
@@ -24,55 +26,7 @@
                 },
                 "apis": [
                     {
-                        "name": "/blocks/latest",
```
Comment (Aleksao998) [inline cookbook/spec_add_lava.json:27]: we cannot delete this one and one after /blocks/{height}

**Review by Yaroms — DISMISSED** [review]

**Review by Yaroms — APPROVED** [review]

**Pull request merged**